src/pybind/mgr/email/module.py

```diff
@@ -0,0 +1,55 @@
+
+"""
+An email module
+
+See doc/mgr/hello.rst for more info.
+"""
+
+from mgr_module import MgrModule
+import smtplib
+
+class Email(MgrModule):
+    COMMANDS = [
+        {
+            "cmd": "Email "
+                   "name=port,type=CephInt,req=false "
+                   "name=host,type=CephString,req=false "
+                   "name=sender_id,type=CephString,req=true "
+                   "name=target_id,type=CephString,req=true "
+                   "name=msg,type=CephString,req=true",
+            "desc": "Sends email",
+            "perm": "rw"
+        },
+    ]
+
+    def handle_command(self, inbuf, cmd):
+        self.log.info("Email_info")
+        self.log.debug("Email_debug")
+        self.log.error("Email_error")
+
+        status_code = 0
+        output_buffer = "Output buffer is for data results"
+        output_string = "Output string is for informative text"
```

Comment:
we don't really need these lines.  on success, we don't need a friendly message.. or can just put it directly in teh return statement below.